### ISSUE TITLE
Leaflet won't export from on Mac without basic settings

### ISSUE THREAD

**Issue by dominijk** [tomchadwin/qgis2web#618]
As further described in my stackoverflow [post](https://gis.stackexchange.com/questions/271997/qgis2web-plugin-leaflet-option-wont-work/272003#272003) . Trying to export as leaflet with anything more than the most basic options results in computer running out of memory, activity monitor shows QGIS to be requiring 24gb and 101% CPU. This is on  16gb memory 2.9Ghx icore 7 machine

**Comment by tomchadwin** [comment]
It's impossible to help from this kind of information, as we need specifics to be able to test. Try to narrow the issue down to a single layer which is causing the problem, and then let us know some details about that layer.

**Comment by tomchadwin** [comment]
OK, I've gone through the tutorial, and I don't seem to have any problems. I didn't do the virtual field bit, though, so I'll try that and see if it makes a difference. I'm on Windows, though, and I don't have access to a Mac to test.

**Comment by tomchadwin** [comment]
I've done the virtual field bit, and it still works for me. I'm not sure what the issue is. Anything in the qgis2web log?

**Comment by dominijk** [comment]
I also skipped the virtual field part, where can i find the log?

**Comment by tomchadwin** [comment]
In QGIS, View > Panels > Log Messages Panel, then click the qgis2web tab.

**Comment by dominijk** [comment]
I'm not seeing that panel, which might be the problem
![screen shot 2018-02-20 at 13 02 01](https://user-images.githubusercontent.com/6041913/36425312-54aea604-163e-11e8-843b-f51a81607f23.png)

**Comment by tomchadwin** [comment]
No, that's OK - if there have been no qgis2web errors, nothing is logged, so the panel doesn't appear.

**Comment by dominijk** [comment]
Ok, so back to the other debug method you suggest to track it

**Comment by dominijk** [comment]
Here's the debug log;

Warning: loading of qgis translation failed [/Applications/QGIS.app/Contents/MacOS/../Resources/i18n//qgis_en_GB]
Warning: loading of qt translation failed [/Developer/Applications/Qt/translations/qt_en_GB]
\Warning: Object::connect: No such slot QgsBookmarks::exportToXML()
Warning: Object::connect:  (receiver name: 'QgsBookmarksBase')
Warning: Object::connect: No such slot QgsBookmarks::importFromXML()
Warning: Object::connect:  (receiver name: 'QgsBookmarksBase')
Warning: QCss::Parser - Failed to load file  "/style.qss" 
Warning: QLayout: Attempting to add QLayout "" to QgsPanelWidgetStack "mWidgetStack", which already has a layout

**Comment by dominijk** [comment]
Here's the log, i had to force quite as mac ran out of memory;
Process 1479 launched: '/Applications/QGIS.app/Contents/MacOS/QGIS' (x86_64)
Warning: loading of qgis translation failed [/Applications/QGIS.app/Contents/MacOS/../Resources/i18n//qgis_en_GB]
Warning: loading of qt translation failed [/Developer/Applications/Qt/translations/qt_en_GB]
Warning: Object::connect: No such slot QgsBookmarks::exportToXML()
Warning: Object::connect: (receiver name: 'QgsBookmarksBase')
Warning: Object::connect: No such slot QgsBookmarks::importFromXML()
Warning: Object::connect: (receiver name: 'QgsBookmarksBase')
Warning: QCss::Parser - Failed to load file "/style.qss"
Warning: QLayout: Attempting to add QLayout "" to QgsPanelWidgetStack "mWidgetStack", which already has a layout
2018-02-23 09:34:00.365 QGIS[1479:69369] modalSession has been exited prematurely - check for a reentrant call to endModalSession:
ne_10m_airports_0
Process 1479 stopped

thread #1: tid = 0x10ef9, 0x0000000117885730 Python_PyString_Resize + 138, queue = 'com.apple.main-thread', stop reason = signal SIGTERM frame #0: 0x0000000117885730 Python_PyString_Resize + 138
Python`_PyString_Resize:
-> 0x117885730 <+138>: movb $0x0, 0x24(%rax,%r14)
0x117885736 <+144>: movq $-0x1, 0x18(%rax)
0x11788573e <+152>: xorl %eax, %eax
0x117885740 <+154>: jmp 0x117885703 ; <+93>

**Comment by dominijk** [comment]
And this is the report i get when i force quite because the process becomes unresponsive
[error.txt](https://github.com/tomchadwin/qgis2web/files/1751330/error.txt)

**Comment by dominijk** [comment]
On my home mac the issue remains. On my work mac (earlier OS) it doesn't
